Example/Controllers/SwiftUI/MiniAppDashboardView.swift

```diff
@@ -23,6 +23,7 @@ struct MiniAppDashboardView: View {
                 }
                 .tabItem {
                     Label("List I", systemImage: "list.bullet")
+                        .accessibilityIdentifier(AccessibilityIdentifiers.tabBarListI)
```

Comment:
@asboxi congratz to raising your first PR 🎉 
one thing to mention is you need to add the identifier at the end of each enum value eg to get the proper full format

```suggestion
.accessibilityIdentifier(AccessibilityIdentifiers.tabBarListI.identifier)
```

Reply:
Thanks for the comment. Fixed!